docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md

```diff
@@ -1,113 +1,93 @@
+1. Click into a room and click the room name or avatar in the top header
+1. Go into Share Code
+1. Screenshot the QR code to add to your deck
 
-### Step 6: Update your rooms throughout the event 
+Add the QR code to every slide so that if folks forget to scan the QR code at the beginning of the presentation, they can still join the discussion.
 
-We find chat to be a powerful way to not only engage your attendees, but direct them in realtime to get exactly where they need to go:
+### Step 4: Plan out your messaging and cadence before the event begins
+Expensify Chat is a great place to provide updates leading up to your event -- share news, get folks excited about speakers, and let attendees know of crucial event information like recommended attire, travel info, and more. For example, you might consider:
 
-- #announce: Use this room to make announcements such as what’s coming up next, where and when social events are taking place, or announcing the sponsor floor is open to the entire conference. Only workspace admins can post in this room.
-- #social: Have your employees in this room sharing fun photos, stoking conversations, and responding to any questions or feedback.
-- Speaker rooms: Encourage your employees to jump in to comment on content and nudge attendees to engage with each other during sessions.
+**Prep your announcements:**
+- Create a document containing drafts of the key messages you intend to send throughout the day.
+- If your event's agenda is broken up into hourly blocks, create a separate section for each hour of the event, to make it easy to find the correct section at the right time.
+- Start each day with a review of the daily agenda, such as a bullet list summarizing what's happening hour by hour.
 
-*Protip*: Expensify Chat has [moderation tools](https://help.expensify.com/articles/other/Everything-About-Chat#flagging-content-as-offensive) to help flag comments deemed to be spam, inconsiderate, intimidating, bullying, harassment, assault. On any comment, just click the flag icon to moderate conversation. 
+**Post your updates:**
+- Designate a team member to post each update in #announce at the designated time.
+- Each hour, send a message listing exactly what is happening next – if there are multiple sessions happening simultaneously, list out each, along with a description of the session, a reminder of where it's located, and (most importantly) a link to the chat room for that session
+- Write the messages in [markdown format](https://help.expensify.com/articles/other/Everything-About-Chat#how-to-format-text), such that they can be copy/pasted directly into Expensify Chat for sending.
+  - If there is some formatting issue upon posting, no problem: just edit the comment after sending, and it'll be fixed for everyone.
+- We’d also recommend posting your updates on new lines so that if someone has a question about a certain item they can ask in a thread pertaining to that topic, rather than in one consolidated block.
```

Comment:
```suggestion
- We’d also recommend posting your updates on individual messages so that if someone has a question about a certain item they can ask in a thread pertaining to that topic, rather than in one consolidated block.
```

NAB: You can do many new lines in a single message and it won't allow you to create separate threads